launch/launcher.go

```diff
@@ -50,10 +53,25 @@ func (l *Launcher) Launch(self string, cmd []string) error {
 		}
 		return nil
 	}
-	launcher, err := l.profileD()
+	profileCmds, err := l.profileD()
 	if err != nil {
 		return errors.Wrap(err, "determine profile")
 	}
+
+	if runtime.GOOS == "windows" {
+		var launcher string
+		if len(profileCmds) > 0 {
+			launcher = strings.Join(profileCmds, " && ") + " && "
+		}
```

Comment:
Joining with " && " should work for both `bash` and `cmd`

Reply:
I'm happy to do this, but we were trying to leave the Linux case rather untouched, as they were joined with newlines before.